install.bat

```diff
@@ -17,15 +17,15 @@ set PYTHONUTF8=1
 :: all dependencies are installed from conda
 set PIP_NO_DEPS=1
 
-set MY_CONDA_ENV_FILE=environments\conda-py-%PY_VER%-win-64.lock.yml
+set MY_CONDA_ENV_FILE=environments\py-%PY_VER%-win-64.conda.lock.yml
 if not exist %MY_CONDA_ENV_FILE% (
   echo "** ERROR: Could not find the conda environment specification file '%MY_CONDA_ENV_FILE%' **"
   pause
   exit /B 0
 )
 
 call "!MY_CONDA!" activate base ^
-  && call "!MY_CONDA!" env create --force -n %ENV_NAME% --file %MY_CONDA_ENV_FILE% ^
+  && call "!MY_CONDA!" env create -y --solver libmamba -n %ENV_NAME% --file %MY_CONDA_ENV_FILE% ^
   && call "!MY_CONDA!" run -n %ENV_NAME% pip install -e .[dash]
```

Comment:
for users, I would not make it editable
```suggestion
  && call "!MY_CONDA!" run -n %ENV_NAME% pip install .[dash]
```

As for developers, we could have a helper script `make-editable.bat` to patch the installation and point to the local source.